infra/modules/providers/azure/app-service/main.tf

```diff
@@ -37,18 +37,19 @@ resource "azurerm_app_service" "appsvc" {
   }
 
   auth_settings {
-    enabled          = local.app_configs[count.index].ad_client_id == "" ? false : true
+    enabled          = false
```

Comment:
Since we're always disabling the auth_settings, are we able to eliminate this block altogether? It's counter intuitive that we're half setting up for the possibility that it *might* be used for authn, but not necessarily required.

Reply:
Removing this and adding the tenant id from the azure cli worked out. Below is a screenshot of visiting a basic web app with authentication.

![image](https://user-images.githubusercontent.com/10041279/63310838-d2a4c080-c2c1-11e9-89c2-16d861962489.png)